File: IndianPad/src/indianwall/vo/FeedsInfo.java
package indianwall.vo;



public class FeedsInfo {

   

       private final String feedCount;

       private final String pageSizeCnt;

       

       public FeedsInfo(String feedCount,String pageSizeCnt) {       

              this.feedCount = feedCount;

              this.pageSizeCnt = pageSizeCnt;

       }



       public String getFeedCount() {

              return feedCount;

       }



       public String getPageSizeCnt() {

              return pageSizeCnt;

       }





       

       

}

File: IndianPad/war/indianwall/reader/js/model/feedsinfo.js
var app = app || {};



app.FeedsInfo = Backbone.Model.extend({

       

       initialize: function() {

              console.log('FeedsInfo is initialized');

       },

       

    defaults: {

       feedCount : 0,

       pageSizeCnt : 0

    },

    

    urlRoot: '/FeedServlet'

       

});
